common-lib/src/main/scala/com/gu/mediaservice/lib/metadata/ImageMetadataConverter.scala

```diff
@@ -52,10 +52,11 @@ object ImageMetadataConverter extends GridLogging {
       res
     }
 
+    val currentDateTime = Some(DateTime.now(DateTimeZone.UTC))  // We expect dateTaken to be before this date.
```

Comment:
By always taking now - this is non-deterministic. If reingested in the future this could change. I think we should use the initial upload date (i.e. the timestamp on the S3 object if it exists). If it is later projected (in 2022 using your example) then the parsing won't change as it will parse it based on when it was first seen.